.github/workflows/test.yml

```
@@ -22,15 +22,15 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: [3.7, 3.8]
+        python-version: ["3.8", "3.9", "3.10"]
```

Why not test recent? Tests should be passing ...
```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11"]
```